spring-cloud-gcp-samples/spring-cloud-gcp-data-multi-sample/src/test/java/com/example/MultipleNamespaceDatastoreIntegrationTest.java

```diff
@@ -42,7 +42,7 @@
 @RunWith(SpringRunner.class)
 @TestPropertySource("classpath:application-test.properties")
 @EnableAutoConfiguration
-public class MultipleNamespaceDatastoreTest {
+public class MultipleNamespaceDatastoreIntegrationTest {
```

Comment:
Tests?

Reply:
Nack'ing these comments - decided in chat that it's not necessary to diff between Test and Tests